configuration/analyzer_config.json

```diff
@@ -1245,6 +1245,35 @@
     },
     "params": {}
   },
+  "GreedyBear": {
+    "type": "observable",
+    "python_module": "greedybear.GreedyBear",
+    "description": "scan an IP against the Greedy Bear API",
```

Comment:
even if the majority of those indicators are IP addresses, this service supports `domain` too. Please add this both in the description and in the `observable_supported`